### PR TITLE
V1.0.2

### PR DESCRIPTION
**Pull request by cliffordp**
https://central.tri.be/issues/101967

### CODE REVIEW

File: src/Tribe/Settings_Helper.php [hunk 1]
```diff
@@ -5,228 +5,222 @@
...
 	/**
-	 * Setup the helper
-	 *
-	 * @param int $priority Priority at which this hooks into 'tribe_settings_tab_fields'.
+	 * Helper for inserting/removing fields on the WP Admin Tribe Settings pages
 	 */
-	public function __construct( $priority = 100 ) {
-		add_filter( 'tribe_settings_tab_fields', array( $this, 'filter_options' ), $priority, 2 );
-	}
+	class Tribe__Extension__Settings_Helper {
```
Comment (barryhughes) [inline src/Tribe/Settings_Helper.php:12]: I _think_ a lot of the changes here are formatting fixes. Can you summarize, though - did any methods change or were any method arguments altered etc?

Comment (cliffordp) [reply src/Tribe/Settings_Helper.php:12]: No substantive Settings Helper changes... just wrapped the whole thing in class_exists() and removed some extra line breaks -- so you're correct

Comment (barryhughes) [reply src/Tribe/Settings_Helper.php:12]: :+1:
